orchagent/portsorch.cpp

```diff
@@ -296,7 +296,21 @@ PortsOrch::PortsOrch(DBConnector *db, vector<table_name_with_pri_t> &tableNames)
     }
 
     m_default1QBridge = attrs[0].value.oid;
-    m_defaultVlan = attrs[1].value.oid;
+    m_defaultVlan.m_vlan_info.vlan_oid = attrs[1].value.oid;
+
+    memset(&attr, 0x00, sizeof(attr));
+    attr.id  = SAI_VLAN_ATTR_VLAN_ID;
+ 
+    /* Get the VLAN ID associated with the default VLAN object */
+    status = sai_vlan_api->get_vlan_attribute(m_defaultVlan.m_vlan_info.vlan_oid, 1, &attr);
+    if (status != SAI_STATUS_SUCCESS)
+    {
+        SWSS_LOG_ERROR("Failed to get default VLAN ID, rv:%d", status);
+        throw runtime_error("PortsOrch initialization failure");
+    }    
+ 
+    m_defaultVlan.m_vlan_info.vlan_id = attr.value.u16;
+
```

Comment:
could you put the default VLAN into the m_portList? Then we don't need to handle special case in the function `addVlan`.